acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/bftsoak/BftMiningSoakTest.java

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright Hyperledger Besu contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.tests.acceptance.bftsoak;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.hyperledger.besu.config.JsonUtil;
+import org.hyperledger.besu.tests.acceptance.bft.BftAcceptanceTestParameterization;
+import org.hyperledger.besu.tests.acceptance.bft.ParameterizedBftTestBase;
+import org.hyperledger.besu.tests.acceptance.dsl.node.BesuNode;
+import org.hyperledger.besu.tests.web3j.generated.SimpleStorage;
+import org.hyperledger.besu.tests.web3j.generated.SimpleStorageShanghai;
+
+import java.math.BigInteger;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class BftMiningSoakTest extends ParameterizedBftTestBase {
+
+  private final int NUM_STEPS = 5;
+
+  private final int MIN_TEST_TIME_MINS = 60;
+
+  private static final long ONE_MINUTE = Duration.of(1, ChronoUnit.MINUTES).toMillis();
+
+  private static final long THREE_MINUTES = Duration.of(1, ChronoUnit.MINUTES).toMillis();
+
+  private static final long TEN_SECONDS = Duration.of(10, ChronoUnit.SECONDS).toMillis();
+
+  static int getTestDurationMins() {
+    // Use a default soak time of 60 mins
+    return Integer.getInteger("acctests.soakTimeMins", 60);
+  }
+
+  @ParameterizedTest(name = "{index}: {0}")
+  @MethodSource("factoryFunctions")
```

Comment:
I went with just re-using the combinations that all the current `BFT` acceptance tests use because I think we might still have users on `IBFT2` with existing chains who still want to move up to `shanghai`. I'd personally suggest that we remove `IBFT2` from all or none (unless we see issues with the run time for this particular test)